codegen/type_converter.go

```diff
@@ -79,6 +81,8 @@ func NewTypeConverter(h PackageNameResolver) *TypeConverter {
 		LineBuilder:   LineBuilder{},
 		Helper:        h,
 		uninitialized: make(map[string]*fieldStruct),
+		convStructMap: make(map[string]string),
+		useRecurGen:   false,
```

Comment:
don't need to init default value for primary types, int=0, bool=false ... etc

Reply:
True, do you want me to remove it? I thought it could be nice to have for clarity